docs/faq.md

```diff
@@ -7,255 +7,89 @@ menuOrder: 5
-the latest version of Serverless Framework and add the `console` property
-to your `serverless.yaml` file.
-
-```yaml
-org: myorg
-console: true
-service: myservice
-frameworkVersion: '3'
-```
-
-**Note:** The following example will disable the use of Serverless Dashboard. 
-
-**What Serverless.yaml fields effect console?**
-
-Several details of your serverless.yaml file will be relevant in Serverless
-Console. This includes, your org (required), stage, service and function names.
-
-```yaml
-
-# Organization name (required)
-org: myorg
-
-# Console flag (required true)
-console: true
+## Adding the AWS  Integration
 
-# App is an optional property that enables Serverless Dashboard integration, which can be used together with Serverless Console
-app: my-app
+**What access do I need to add an AWS Account?**
+You will need to admin access to setup the AWS Account. Adding the Integration adds a CloudFormation stack that configures an IAM Role that then creates additional resources. More details are available in the [adding the AWS Integration guide](../integrations/index.md)
 
-# Disable all Dashboard features aside of provider credentials and parameters retrieval
-dashboard:
-  disableMonitoring: true
+**What infrastructure is getting setup in my AWS Account?**
+When you add the AWS Integration the following components get setup.
 
-# Service name (required)
-service: myservice
+* [CloudFormation Stack](./glossary.md#cloudformation-stack)
+* [IAM Role](./glossary.md#iam-roles)
+* [Metric Streams](./glossary.md#cloudwatch-metric-stream)
+* [EventBridge rules](./glossary.md#eventbridge)
+* [CloudTrail trail](./glossary.md#cloudtrail)
 
-# Framework version 3.7.1 or higher
-frameworkVersion: '3.7.1'
+After you have added the AWS Observability integration, [enabling additional monitoring features](./integrations/enable-monitoring-features.md) will also create.
 
-# Function names are provided as filters within Console	
-functions:
-  hello:
+* [Lambda Layers](./glossary.md#lambda-layer)
+* [Cloudwatch Log Subscription Filters](./glossary.md#cloudwatch-subscriptions)
 
-```
+**How do I delete infrastructure setup by Serverless Console?**
+The best way to delete the infrastructure provisioned by Serverless Console is to Remove the integration using the Console UI. This will remove all the infrastructure associated with Serverless Console. 
 
-Additionally the stage parameter will be included as an environment filter within
-Serverless Console.
+If you wish to disable function specific [Lambda Layers](./glossary.md#lambda-layer) or [Cloudwatch Log Subscription Filters](./glossary.md#cloudwatch-subscriptions) you can [disable function specific monitoring features](./integrations/enable-monitoring-features.md) on the integrations page.
 
-```text
-serverless deploy --stage #included as environment
-```
+If you do need to delete manually check our [guide for removing the AWS Integration](./integrations/removing-aws-integration.md)
```

Comment:
`Follow the [guide for removing the AWS Integration](./integrations/removing-aws-integration.md) if you need to manually remove the AWS Integration`.